packages/desktop-client/src/components/reports/graphs/net-worth-spreadsheet.tsx

```diff
@@ -11,9 +9,33 @@ import {
   amountToInteger,
 } from 'loot-core/src/shared/util';
 
-import AlignedText from '../../common/AlignedText';
 import { index } from '../util';
 
+function formatDate(date) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could you please re-use the existing date formatters instead of building a new one?

https://github.com/actualbudget/actual/blob/master/packages/loot-core/src/shared/months.ts

Reply:
done